Core/Frameworks/BaikalAdmin/Controller/Install/Initialize.php

```diff
@@ -101,13 +98,12 @@ function render() {
         // we need to tell the installer page to show a warning message.
         $oView->setData("oldConfigSystem", file_exists(PROJECT_PATH_SPECIFIC . "config.system.php"));
 
-
         if ($this->oForm->persisted()) {
             $sLink = PROJECT_URI . "admin/install/?/database";
             \Flake\Util\Tools::redirect($sLink);
             exit(0);
 
-            #$sMessage = "<p>Baïkal is now configured. You may <a class='btn btn-success' href='" . PROJECT_URI . "admin/'>Access the Baïkal admin</a></p>";
+        #$sMessage = "<p>Baïkal is now configured. You may <a class='btn btn-success' href='" . PROJECT_URI . "admin/'>Access the Baïkal admin</a></p>";
```

Comment:
The code-style-checker really wants to "fix" this indent.

Reply:
Noticed that, too. Maybe we could simply remove that comment.